### ISSUE TITLE
Commit Buffer Error in SWHS_SRS.tex

### ISSUE THREAD

**Issue by samm82** [JacquesCarette/Drasil#659]
In master, running `make` on my system gives the following error (noticed when Travis build failed on some import branches):

```
cd build/SWHS && stack exec swhs
swhs.EXE: SRS/SWHS_SRS.tex: commitBuffer: invalid argument (invalid character)
make: *** [swhs_build] Error 1
```

@elwazana and @niazim3 cannot replicate this issue; it only happens on my machine.

**Comment by smiths** [comment]
@samm82, I couldn't replicate your error either.  Maybe try make clean first.  Sometimes there are bits of "stuff" left over that aren't properly built.  make clean means you are starting from a clean slate.

**Comment by samm82** [comment]
@smiths I did

**Comment by samm82** [comment]
Running `make` in Windows command prompt after following the guidelines of #482 works like a charm. I'm not sure why this is coming up now, but I guess I'll just have to use command prompt for `make`.

**Comment by bmaclach** [comment]
I had this same issue running `make` in Git Bash and was able to fix it by running `chcp.com 65001` before running `make`, all in Git Bash. This is slightly different than the fix outlined in #482 so I am documenting it here in case anyone else runs into this issue.

**Comment by smiths** [comment]
Welcome back @bmaclach.  😄   Thank you for making a record of your fix.
